app/controllers/api/v1x0.rb

```diff
@@ -0,0 +1,97 @@
+module Api
+  module V1x0
+    class RootController < ApplicationController
+      def openapi
+        render :json => Api::Docs["1.0"].content.to_json
+      end
+    end
+
+    class AuthenticationsController < Api::V0x0::AuthenticationsController
+      include Api::V0x1::Mixins::IndexMixin
```

Comment:
@abellotti could we also just drop `v0.0` and move the `v0x1` index_mixin into the base mixins?

Reply:
no one is using /api/v0.0  (just the /internal/v0.0), should I drop that in this PR ?

Reply:
ok, /api/v0.0 goes bye bye